python/lsst/consdb/pqserver.py

```diff
@@ -24,7 +24,7 @@
 import sqlalchemy
 import sqlalchemy.dialects.postgresql
 from flask import Flask, request
-from utils import setup_logging, setup_postgres
+from .utils import setup_logging, setup_postgres
```

Comment:
I think this doesn't work in the Docker container after copying the files.  We may need to reorganize the destination tree to make this function properly.